### DISCUSSION TITLE
Bookmarks Not Switching to Playback

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1156] [Bug Reports]
@rern 
The radio bookmarks now follow the Replace and Play setting (thanks), but they ignore the Switch To Playback option.

Sometimes the station selected does not play or show metadata, even when not using the bookmark. This problem looks random so far, and only since 20221221 update.

**Comment by Raitsa** [comment]
Confirmed, noticed right after updating to 20221221.

**Comment by hjfisch** [comment]
Same for me . But now I'm booting  my RPI 3 from USB and suddenly all my probs are gone . Switching to playback is working and most of all , building a new database , which took hours before , takes only a few minutes . Maybe it is because I was using an extension-cable for my sd-card mounted to the housing . Now my player is working as expected.
